hw/ip/csrng/data/csrng.hjson

```diff
@@ -592,6 +594,8 @@
       hwaccess: "hro",
       hwqe: "true",
       regwen: "REGWEN",
+      tags: [// Internal HW can modify status register
```

Comment:
this comment does not seem right, since this is an error code test register.

Reply:
updated comment to reflect not wanted a fatal alert.